src/Util/Lock.php

```diff
@@ -44,6 +46,25 @@ private static function connectMemcache() {
 	}
 
 	/**
+	 * @brief Creates a semaphore key
+	 *
+	 * @param string $fn_name Name of the lock
+	 *
+	 * @return ressource the semaphore key
+	 */
+	private static function semaphore_key($fn_name) {
```

Comment:
The names of the methods in classes should be camelCase.

Ideally the method name should begin with a verb which describes what the method is doing. I know you like short function names to don't have to write too much but since it isn't used often, I think we can name the method `createSemaphoreKey`